WordPress/WordPressTest/MediaPicker/MediaLibraryPickerDataSourceTests.swift

```diff
@@ -27,10 +27,30 @@ class MediaLibraryPickerDataSourceTests: XCTestCase {
             }
             newMedia = media
         })
-        self.waitForExpectations(timeout: 2000, handler: nil)
+        self.waitForExpectations(timeout: 5, handler: nil)
         return newMedia
     }
 
+    fileprivate func newVideoMedia() -> Media? {
```

Comment:
I think the `testImage` and `testVideo` tests look like they could be split into a couple of individual tests (with more descriptive names). For example `testImageRequestedWithZeroSize`, `testImageRequestedWithSpecificSize`, `testImageRequestedFromVideoMedia`...